website/src/docs/hotchocolate/v14/fetching-data/dataloader.md

```diff
@@ -113,13 +113,13 @@ It executes resolvers until the queue is empty and then triggers the data loader
 
 # Data Consistency
 
-Dataloader do not only batch calls to the database, they also cache the database response.
+DataLoader do not only batch calls to the database, they also cache the database response.
 A data loader guarantees data consistency in a single request.
 If you load an entity with a data loader in your request more than once, it is given that these two entities are equivalent.
 
 Data loaders do not fetch an entity if there is already an entity with the requested key in the cache.
 
-# Types of Data loaders
+# Types of DataLoader
```

Comment:
It could be argued that this is conceptually referring to a "Data Loader" thing, but not THE `DataLoader` identifier.  Either way, should be plural.